api/src/main/java/io/opentelemetry/metrics/InstrumentWithBind.java

```diff
@@ -37,12 +37,15 @@
    */
   B bind(LabelSet labelSet);
 
-  /**
-   * Removes the {@code Bound Instrument} from the Instrument. i.e. references to previous {@code
-   * Bound Instrument} are invalid (not being managed by the instrument).
-   *
-   * @param boundInstrument the {@code Bound Instrument} to be removed.
-   * @since 0.1.0
-   */
-  void unbind(B boundInstrument);
+  interface Bound {
```

Comment:
I think this should be called "Binding"

Reply:
I am confused about this strange language called English :))

Reply:
🤣  yeah, it's a pretty crazy language

Reply:
Went with `BoundInstrument`, does that work?

Reply:
love it. ❤️